assets/src/data/eventespresso/core/actions/receive-entities.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Internal imports
+ */
+import { ACTION_TYPES } from './action-types';
+const { entities: types } = ACTION_TYPES;
```

Comment:
tomato, tomahto :) I don't think its inconsistent - inconsistent with what exactly?  

It's an intentional choice here for a couple reasons:

- `types` is an object containing type constants.
- In the IDE as you type `types` it'll auto-complete to the properties available on it.  

Sure, one could prefix `TYPE` onto each constant but to me that seems a bit redundant because its already exported as `ACTION_TYPES`... so `ACTION_TYPES.entities.TYPE_...`?  Redundant.